src/components/features/price-filter/price-filter.tsx

```
@@ -0,0 +1,105 @@
+import { useEffect, useState } from "react";
+
+export default function PriceFilter() {
+  const t = useTranslations("Price");
+  const { watch, setValue } = useFormContext();
+
+  // Watch Price inputs
+  const priceFrom = watch("priceFrom");
+  const priceTo = watch("priceTo");
+
+  // Requested Products state
+  const [products, setProducts] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)